spacy/tests/doc/test_morphanalysis.py

```diff
@@ -32,7 +32,8 @@ def test_token_morph_key(i_has):
 
 def test_morph_props(i_has):
     assert i_has[0].morph.get("PronType") == ["prs"]
-    assert i_has[1].morph.get("PronType") == []
```

Comment:
Why did you remove the original assert, I thought that behaviour would still be valid?

Reply:
No reason, I'll add it back.